dist/nix/vim-with-quick-lint-js.nix

```diff
@@ -0,0 +1,14 @@
+{ pkgs ? import <nixpkgs> {} 
+, vimrcWithPlugins ? import ./vimrc-with-plugins.nix { inherit pkgs; } 
+, vim  ? pkgs.vim #Pick your vim
+, name ? "jsvim"
+}:
+pkgs.symlinkJoin {
+  inherit name;
+  paths = [ (pkgs.vim_configurable.override { python = pkgs.python3; }) ];
```

Comment:
Why do we need Python 3 support in Vim?

Reply:
the idea of this file is to be an example file, you we dont need it.